### DISCUSSION TITLE
How interested would users be in a new package `@remirror/extension-canvas-draw`?

### DISCUSSION THREAD

**Discussion by ifiokjr** [remirror/remirror#76] [Ideas]
## Detailed Description

I saw this [tweet today](https://twitter.com/juliandoesstuff/status/1114172547745161216) which got me thinking, how about adding an extension which enables canvas doodles to be added to the editor.

![image](https://user-images.githubusercontent.com/1160934/55646304-bac91600-57d2-11e9-9a33-fc8db54a6b1a.png)

## Possible Implementation

The [codesandbox](https://codesandbox.io/s/github/juliankrispel/draft-js-with-canvas-block
) and [blog](https://reactrocket.com/post/draft-js-and-canvas/) should be useful guides.

**Comment by Bertg** [comment]
👍
